### DISCUSSION TITLE
Issue on deployment https://github.com/pnp/sp-dev-fx-extensions/tree/master/samples/react-application-announcements

### DISCUSSION THREAD

**Discussion by JamesMil** [pnp/sp-dev-fx-extensions#355] [General]
Receiving the attached error, any help appreciated
![Error](https://user-images.githubusercontent.com/69344581/106451275-399a4300-647e-11eb-82e0-ff450fb7deef.png)

Error:
Add-PnPCustomAction : A positional parameter cannot be found that accepts argument
'-ClientSideComponentProperties={&quot;listName&quot;:&quot;Site Announcements&quot;,
&quot;siteUrl&quot;:&quot;/sites/Home-Test&quot;}'.
At line:1 char:1
+ Add-PnPCustomAction -Name "Announcements" -Title "Announcements" -Loc ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Add-PnPCustomAction], ParameterBindingException
    + FullyQualifiedErrorId : PositionalParameterNotFound,PnP.PowerShell.Commands.Branding.AddCustomAction

**Comment by AriGunawan** [comment]
Hi @JamesMil,
I think there shouldn't be a `=` after the `-ClientSideComponentProperties`

```diff
Add-PnPCustomAction -Name "Announcements" `
   -Title "Announcements" `
   -Location "ClientSideExtension.ApplicationCustomizer" `
   -ClientSideComponentId "dd7ec4fd-97aa-44c5-b6ad-87535862e0bf" 
-  -ClientSideComponentProperties="{&quot;listName&quot;:&quot;Site Announcements&quot;, &quot;siteUrl&quot;:&quot;/sites/Contoso&quot;}"
+  -ClientSideComponentProperties "{&quot;listName&quot;:&quot;Site Announcements&quot;, &quot;siteUrl&quot;:&quot;/sites/Contoso&quot;}"
```

Could you try execute below command:
```cmd

Add-PnPCustomAction -Name "Announcements" -Title "Announcements" -Location "ClientSideExtension.ApplicationCustomizer" -ClientSideComponentId "dd7ec4fd-97aa-44c5-b6ad-87535862e0bf" -ClientSideComponentProperties "{&quot;listName&quot;:&quot;Site Announcements&quot;, &quot;siteUrl&quot;:&quot;/sites/Contoso&quot;}"
```
